dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
+curl -s "${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME" | jq '.status.hostIP' | sed 's/"//g'
```

Comment:
@adleong [kubernetes #24657](https://github.com/kubernetes/kubernetes/issues/24657) just make a proposal for hostIP which is finally merged in [kubernetes #42717](https://github.com/kubernetes/kubernetes/issues/42717), in [kubernetes #27880](https://github.com/kubernetes/kubernetes/issues/27880), `spec.NodeName` is added to downward API.